aptos-move/framework/aptos-stdlib/sources/cryptography/groups.move

```diff
@@ -0,0 +1,751 @@
+module aptos_std::groups {
+    use std::option::Option;
+
+    // Error codes
+    const E_NATIVE_FUN_NOT_AVAILABLE: u64 = 1;
+    const E_UNKNOWN_GROUP: u64 = 2;
+    const E_UNKNOWN_PAIRING: u64 = 3;
+
+    /// `BLS12_381_G1` represents a group used in BLS12-381 pairing.
+    /// The group is a prime-order group on an elliptic curve `y^2=x^3+4` defined over `Fq`.
+    /// `Fq` is a finite field with `q=0x1a0111ea397fe69a4b1ba7b6434bacd764774b84f38512bf6730d2a0f6b0f6241eabfffeb153ffffb9feffffffffaaab`.
+    /// THe order of the group `r` is 0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001.
```

Comment:
Typo: **The** not "THe"

Also, clarify if this is the non-prime order order of the whole elliptic curve, or the prime-order of the G1 subgroup?

Remember that we will only support the prime order subgroup for now, so this text should probably refer to the prime order.

You can make a passing note about the order of the whole elliptic curve, if you want.

Reply:
I can see that it's the prime order (from [here](https://www.wolframalpha.com/input?i=0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001+prime)). So perhaps clarify by explicitly saying "The **prime** order of the group `r` ..."